Dockerfile

```diff
@@ -0,0 +1,102 @@
+FROM centos:centos7.4.1708
+
+ENV GOSU_VERSION 1.10
+ENV GOSU_ARCH amd64
+ENV GOSU_URL https://github.com/tianon/gosu/releases/download
+ENV GOSU_APP ${GOSU_URL}/${GOSU_VERSION}/gosu-${GOSU_ARCH}
+ENV GOSU_ASC ${GOSU_URL}/${GOSU_VERSION}/gosu-${GOSU_ARCH}.asc
+ENV DB4_VERSION 4.8.30.NC
+ENV DB4_HASH 12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef
+ENV DB4_URL http://download.oracle.com/berkeley-db/db-${DB4_VERSION}.tar.gz
```

Comment:
Wouldn't it make more sense to only have a minimum set of build dependencies and then build the Bitcoin Core dependencies from `./depends/`?